api/Clipboard.json

```diff
@@ -29,10 +29,10 @@
             "version_added": null
```

Comment:
FF doesn't currently support it:
https://dxr.mozilla.org/mozilla-central/rev/2318adaec61f07eccf66c3f742497aa973b15f2f/testing/web-platform/meta/clipboard-apis/async-interfaces.https.html.ini#23-36

Reply:
Fixed in d18c841a4313afbc91ba9eeabb05124c2ad66b53.